src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/OracleTranslator.java

```diff
@@ -294,6 +297,29 @@ public String translate(DbColumn c) {
         }
     }
 
+    @Override
+    public String translate(final DbColumnType type) {
+        switch (type) {
+            case BOOLEAN:
+                return "NUMBER(1,0)";
```

Comment:
We need to keep the types as compatible as possible with the `translate(DbColumn c)` because we will have PDB trying to interpret them.

That said, shouldn't this be `char`?

Reply:
I think it's debatable. For what I saw in the community, there's not a consensus about it. Some people prefer a char, like 't'/'f', others believe that a "BIT" it's better.

I honestly don't like representing things with text, unless I have to. Why would it be better to be a `char`?